app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,35 @@
+class ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventCatcher::Stream
+
+  attr_reader :ems, :stop_polling, :poll_sleep
+
+  def initialize(ems, options = {})
+    @ems = ems
+    @stop_polling = false
+    @from = DateTime.current.to_time.to_i.to_s
+    @poll_sleep = options[:poll_sleep] || 20.seconds
+  end
+
+  def start
+    @stop_polling = false
+    @from = DateTime.current.to_time.to_i.to_s
+  end
+
+  def stop
+    @stop_polling = true
+  end
+
+  def poll
+    loop do
+      @to = DateTime.current.to_time.to_i.to_s
+
+      ems.with_provider_connection(:service => 'ActivityTrackerApiV1', :provider_region => 'us-south') do |api|
```

Comment:
The region that is being passed in here is the region in which the Activity Tracker instance is located in. If there is a better way to go about this please let me know

Reply:
Oh, do you know if the Activity Tracking isn't available on each region?  Usually our provider is added for a specific region so you can just use that.

Reply:
It is available in the following regions: https://cloud.ibm.com/docs/activity-tracker?topic=activity-tracker-endpoints, which I think is all the regions? In that case should we assume that the activity tracker instance is provisioned in the same region as the provider?